index.html

```diff
@@ -158,11 +158,11 @@ <h2 class="project-tagline">The Homebrew package manager for Linux</h2>
 <h2>
 <a id="contributing" class="anchor" href="#contributing" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Contributing</h2>
 
-<p>We'd love you to contribute to Linuxbrew or its upstream, Homebrew. First, please read our <a href="https://github.com/Linuxbrew/brew/blob/master/.github/CONTRIBUTING.md">Contribution Guide</a> and <a href="https://github.com/Linuxbrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct">Code of Conduct</a>.</p>
+<p>We'd love you to contribute to Linuxbrew or its upstream, Homebrew. First, please read our <a href="https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md">Contribution Guide</a> and <a href="https://github.com/Linuxbrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct">Code of Conduct</a>.</p>
 
 <p>We explicitly welcome contributions from people who have never contributed to open-source before: we were all beginners once! We can help build on a partially working pull request with the aim of getting it merged. We are also actively seeking to diversify our contributors and especially welcome contributions from women from all backgrounds and people of colour.</p>
 
-<p>Please see our <a href="https://github.com/Linuxbrew/brew/blob/master/.github/CONTRIBUTING.md#contributing-to-linuxbrew">guidelines</a> on whether to send pull requests to Linuxbrew or Homebrew.</p>
+<p>Please see our <a href="https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md#contributing-to-linuxbrew">guidelines</a> on whether to send pull requests to Linuxbrew or Homebrew.</p>
 
 <p>A good starting point for contributing is running <code>brew audit</code> (or <code>brew audit --strict</code>) with some of the packages you use (e.g. <code>brew audit wget</code> if you use <code>wget</code>) and then read through the warnings, try to fix them until <code>brew audit</code> shows no results and <a href="https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md">submit a pull request</a>. If no formulae you use have warnings you can run <code>brew audit</code> without arguments to have it run on all packages and pick one. Good luck!</p>
```

Comment:
While you're at it, can you also change https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md to https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md ?